python_modules/dagster/dagster/_core/definitions/cacheable_assets.py

```diff
@@ -18,6 +18,14 @@
 from dagster._serdes import whitelist_for_serdes
 from dagster._utils import hash_collection
 
+# Cached data can be stored directly on AssetsDefinitions passed to a repository/Definitions
+# object. Any asset which has CACHED_ASSET_ID_KEY specified will have its metadata stored
+# in the cache under the key CACHED_ASSET_PREFIX + id. This allows for the metadata
+# to be retrieved in subsequent loads.
+CACHED_ASSET_ID_KEY = "dagster/cached_asset_id"
```

Comment:
Right now I think we don't hide period-prefixed metadata, only tags

Reply:
When you say "the cache", what exactly is "the cache" and how does one access it? Wondering if it would make sense to choose a more specific name that alludes to the answers to those questions.

Reply:
Updated to the perhaps overly verbose `dagster/repo_load_data_cached_asset_metadata` for now, which hopefully helps answer that question.

When you say "how does one access it", do we expect these names to be clear to a typical end user, or just clear to an integration author touching these apis?